kof-operator/webapp/collector/src/components/pages/dashboards/DashboardList.tsx

```diff
@@ -26,6 +26,9 @@ const DashboardList = <Items extends K8sObjectSet<Item>, Item extends K8sObject>
 }: DashboardData<Items, Item>): JSX.Element => {
   const { items } = store();
   const navigate = useNavigate();
+  const getPath = (item: Item): string => {
+    return !item.namespace ? item.name : `${item.namespace}/${item.name}`;
```

Comment:
The same:
```suggestion
    return item.namespace ? `${item.namespace}/${item.name}` : item.name;
```
Also, could this function be moved and reused in the suggestion above?